reflex/utils/format.py

```diff
@@ -381,7 +381,7 @@ def format_image_data(value: Type) -> str:
     return f"data:image/png;base64,{base64_image}"
 
 
-def format_state(value: Any) -> Dict:
+def format_state(value: Any) -> Any:
```

Comment:
I updated the type here since if a string is passed into format
-state.py then the return Type is also string.